Formula/redland.rb

```diff
@@ -3,6 +3,7 @@ class Redland < Formula
   homepage "https://librdf.org/"
   url "https://download.librdf.org/source/redland-1.0.17.tar.gz"
   sha256 "de1847f7b59021c16bdc72abb4d8e2d9187cd6124d69156f3326dd34ee043681"
+  license any_of: ["LGPL-2.1-or-later", "GPL-2.0-or-later", "Apache-2.0"]
```

Comment:
LICENSE.txt:
```
                    Redland librdf RDF API Library - License

   This package is Free Software available under any one of the specified
   licenses below. All the licenses below are alternatives and if you
   select one license, that one alone applies.

1. The GNU Lesser General Public License (LGPL) Version 2.1 or any newer version

   See [1]http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html or
   [2]COPYING.LIB for the full license text.
     __________________________________________________________________

     Copyright (C) 2000-2012 [3]David Beckett
     Copyright (C) 2000-2005 [4]University of Bristol. All Rights
     Reserved.

     This package is free software; you can redistribute it and/or modify
     it under the terms of the GNU Lesser General Public License Version
     2.1 as published by the Free Software Foundation or any newer
     version.

     This package is distributed in the hope that it will be useful, but
     WITHOUT ANY WARRANTY; without even the implied warranty of
     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
     Lesser General Public License Version 2.1 for more details.

     You should have received a copy of the GNU Lesser General Public
     License Version 2.1 along with this package; if not, write to the
     Free Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston,
     MA 02110-1301 USA
     __________________________________________________________________

2. GNU General Public License (GPL) V2 or any newer version

   Under Term 3 of the LGPL Version 2.1, you may choose to license the
   entire package under the GPL. If that option is chosen, then this
   package is licensed under the terms of the GPL Version 2 or
   alternatively, any newer version of the GPL. See [5]COPYING for the
   full GPL license text.

3. The Apache License V2.0 or any newer version

   See [6]http://www.apache.org/licenses/LICENSE-2.0 or [7]LICENSE-2.0.txt
   for the full ASL 2.0 license text.

     Copyright (C) 2000-2012 [8]David Beckett
     Copyright (C) 2000-2005 [9]University of Bristol.

     Licensed under the Apache License, Version 2.0 (the "License"); you
     may not use this file except in compliance with the License. You may
     obtain a copy of the License at:

     [10]http://www.apache.org/licenses/LICENSE-2.0

     Unless required by applicable law or agreed to in writing, software
     distributed under the License is distributed on an "AS IS" BASIS,
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
     implied. See the License for the specific language governing
     permissions and limitations under the License.

   The [11]NOTICE file contains the notices that must be applied according
   to section 4(d) of the Apache License, Version 2.0.
     __________________________________________________________________

   Copyright (C) 2000-2013 [12]Dave Beckett
   Copyright (C) 2000-2005 [13]University of Bristol
```